app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great job on setting up the login page UI however it would be better if you provide `sign-up` link on the login page so that a new user can register into the app if they are not already `signed in`. 

<img width="1512" alt="Screenshot 2023-04-06 at 10 29 36 AM" src="https://user-images.githubusercontent.com/44798044/230280214-d97ab1e0-fb25-4d18-888c-996bd114dc95.png">